typescript/infra/src/utils/timelock.ts

```diff
@@ -0,0 +1,100 @@
+import { TimelockController__factory } from '@hyperlane-xyz/core';
+import {
+  ChainMap,
+  ChainName,
+  EXECUTOR_ROLE,
+  MultiProvider,
+  PROPOSER_ROLE,
+  TimelockConfig,
+} from '@hyperlane-xyz/sdk';
+import { Address, assert, rootLogger } from '@hyperlane-xyz/utils';
+
+import { DEPLOYER } from '../../config/environments/mainnet3/owners.js';
+
+export const DEFAULT_TIMELOCK_DELAY = 1;
```

Comment:
We need to remember to update this ahahahah

Reply:
Also might be a goog idea to rename it to `DEFAULT_TIMELOCK_DELAY_IN_SECONDS` just to be extra clear

Reply:
updated to 1 day for now to be safe